locales/base/translation.json

```diff
@@ -590,8 +590,8 @@
   },
   "sendAndInvite": "Send & Invite",
   "inviteAndSendDialog": {
-    "title": "We'll hold on to your payment to {{name}}",
-    "body": "Share an invitation message with {{name}} to join Valora. Once they join & connect a phone number, they'll receive your payment.",
+    "title": "Finish inviting {{name}} to Valora",
+    "body": "Once they join & connect a phone number, you both receive {{amount}} {{currency}}. Click below to send and customize your invite.",
```

Comment:
Would this roll out to everyone when this version of the app is released? Wonder if this message should be behind a flag and only go out to US users, since the rewards program is for US users only? cc @kathaypacific

Reply:
yeah i'm looking at this change and a bit confused - this one is for all users :( this is also a breaking change for older versions of the app once the OTA translations are updated since it contains `amount` and `currency` which weren't there before

Reply:
the only copy change we agreed to was for the prefilled SMS - there is different copy based on the feature flag there, but this one wasn't in scope 🙈 looks like we want this change though, so i'll update with the remote config/make it a non breaking change then create a new patch release